dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/TransferFunction.mo

```diff
@@ -75,7 +67,7 @@ form. Internally, vector <strong>x</strong> is scaled to improve the numerics (t
 not visible from the outside of this block because the non-scaled vector <strong>x</strong>
 is provided as output signal and the start value is with respect to the non-scaled
 vector <strong>x</strong>.
-Initial values of the states <strong>x</strong> can be set via parameter <strong>x_start</strong>.</p><p>If the highest-order component of the denominator is zero, the block returns the input as output.</p>
+Initial values of the states <strong>x</strong> can be set via parameter <strong>x_start</strong>.</p><p>The denominator a has to be at least of order 1 i.e. of size 2.</p><p>If the highest-order component of the denominator is zero, the block returns the input as output.</p>
```

Comment:
Just add more explanation about the difference with the TransferFunction model of MSL and the utility for Dynawo simulations.

Reply:
Explanation added